File: mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("Height (Between 1 and 8): ");
    }
    while (height < 1 || height > 8);

    // For Loop represent each row in the pyramid
    for (int i = 1; i <= height; i++)
    {

        // Two for Loop's used to print the spaces in the inverted pyramid and the following #'s
        for (int k = height; k > i; k--)
        {
            printf(" ");
        }
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }

        printf("  ");

        // For Loop to print out the second pyramid after the two spaces
        for (int z = 1; z <= i; z++)
        {
            printf("#");
        }

        printf("\n");
    }

}

File: sentimental-mario-more/mario.py
# python mario.py
from cs50 import get_int

# Get pyramid height from the user
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

# Iterate through each row of the pyramid
for i in range(1, height+1):

    # First two for loops print out the inverted pyramid
    for k in range(i, height):
        print(" ", end="")

    for j in range(1, i+1):
        print("#", end="")

    # Separte the inverted pyramid and the normal pyramid on the same line
    print("  ", end="")

    # Print out the normal pyramid
    for z in range(1, i+1):
        print("#", end="")

    # Insert new line for next row
    print("")
